CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-x.x.x
+2.0.0
 ------
 
-* Support Django 1.11
+* Upgraded to Python 3
```

Comment:
Maybe we should break this into two updates, so that everything up to "Upgrade to Python 3" is on some 1.x.x version and then "Upgraded to Python 3" is on 2.0.0? Otherwise, it looks odd to see "Changed the base image in Dockerfile from python:2.7 to python:2.7-stretch" in there.

Reply:
well...makes sense to me. Do we want to keep all the changes made, except Python 3 upgrade in 1.2.5 version, where we only got rid of empty except to get rid of flake8 errors and have the Python 3 upgrade in 2.0.0?

Reply:
I think splitting it that way makes sense to me